pipeline/outputs/influxdb.md

```diff
@@ -191,6 +192,43 @@ pipeline:
 {% endtab %}
 {% endtabs %}
 
+### Prefix stripping
+
+When collecting data from many inputs into many buckets, it can be helpful to remove a common prefix using `Strip_prefix`.
+
+```python
```

Comment:
I think we should be using the recently made default approach of having both configs (YAML and legacy) in place.